SimPEG/Examples/DC_PseudoSection_Simulation.py

```diff
@@ -0,0 +1,220 @@
+    """
+
+        DC Forward Simulation
+        =====================
+
+        Forward model conductive spheres in a half-space and plot a pseudo-section
+
+        Created on Mon Feb 01 19:28:06 2016
+
+        @fourndo
+
+    """
+
+    def getIndicesSphere(center,radius,ccMesh):
+        """
+            Creates a vector containing the sphere indices in the cell centers mesh.
+            Returns a tuple
+
+            The sphere is defined by the points
+
+            p0, describe the position of the center of the cell
+
+            r, describe the radius of the sphere.
+
+            ccMesh represents the cell-centered mesh
+
+            The points p0 must live in the the same dimensional space as the mesh.
+
+        """
+
+        # Validation: mesh and point (p0) live in the same dimensional space
+        dimMesh = np.size(ccMesh[0,:])
+        assert len(center) == dimMesh, "Dimension mismatch. len(p0) != dimMesh"
+
+        if dimMesh == 1:
+           # Define the reference points
+
+            ind  = np.abs(center[0] - ccMesh[:,0]) < radius
+
+        elif dimMesh == 2:
+           # Define the reference points
+
+            ind = np.sqrt( ( center[0] - ccMesh[:,0] )**2 + ( center[1] - ccMesh[:,1] )**2 ) < radius
+
+        elif dimMesh == 3:
+            # Define the points
+            ind = np.sqrt( ( center[0] - ccMesh[:,0] )**2 + ( center[1] - ccMesh[:,1] )**2 + ( center[2] - ccMesh[:,2] )**2 ) < radius
+
+        # Return a tuple
+        return ind
+    # First we need to create a mesh and a model.
+
+    # This is our mesh
+    dx    = 5.
+
+    hxind = [(dx,15,-1.3), (dx, 75), (dx,15,1.3)]
+    hyind = [(dx,15,-1.3), (dx, 10), (dx,15,1.3)]
+    hzind = [(dx,15,-1.3),(dx, 15)]
+
+    mesh = Mesh.TensorMesh([hxind, hyind, hzind], 'CCN')
+
+
+    # Set background conductivity
+    model = np.ones(mesh.nC) * sig[0]
+
+    # First anomaly
+    ind = getIndicesSphere(loc[:,0],radi[0],mesh.gridCC)
+    model[ind] = sig[1]
+
+    # Second anomaly
+    ind = getIndicesSphere(loc[:,1],radi[1],mesh.gridCC)
+    model[ind] = sig[2]
+
+    # Get index of the center
+    indy = int(mesh.nCy/2)
+
+
+    # Plot the model for reference
+    # Define core mesh extent
+    xlim = 200
+    zlim = 125
+
+    # Specify the survey type: "pdp" | "dpdp"
+
+
+    # Then specify the end points of the survey. Let's keep it simple for now and survey above the anomalies, top of the mesh
+    ends = [(-175,0),(175,0)]
+    ends = np.c_[np.asarray(ends),np.ones(2).T*mesh.vectorNz[-1]]
+
+    # Snap the endpoints to the grid. Easier to create 2D section.
+    indx = Utils.closestPoints(mesh, ends )
+    locs = np.c_[mesh.gridCC[indx,0],mesh.gridCC[indx,1],np.ones(2).T*mesh.vectorNz[-1]]
+
+    # We will handle the geometry of the survey for you and create all the combination of tx-rx along line
+    [Tx, Rx] = DC.gen_DCIPsurvey(locs, mesh, stype, param[0], param[1], param[2])
```

Comment:
This is erroring  at the moment

<img width="581" alt="examples lindseyjh lindseys-macbook-pro-2 -zsh 161x28 2016-04-07 07-14-54" src="https://cloud.githubusercontent.com/assets/6361812/14354179/3172e330-fc91-11e5-976c-bddc70538b5a.png">